packages/cli/bin/run.cmd

```diff
@@ -1,3 +0,0 @@
-@echo off
-
-node "%~dp0\run" %*
```

Comment:
In case you're wondering, npm automatically generates the cmd version for Windows when you install the software.